WordPress/Classes/Networking/BlogServiceRemoteXMLRPC.m

```diff
@@ -104,66 +101,62 @@ - (void)updateBlogSettings:(RemoteBlogSettings *)remoteBlogSettings
 
 
 
-- (WPXMLRPCRequestOperation *)operationForOptionsWithSuccess:(OptionsHandler)success
-                                                    failure:(void (^)(NSError *error))failure
+- (void)operationForOptionsWithSuccess:(OptionsHandler)success
+                               failure:(void (^)(NSError *error))failure
```

Comment:
If this doesn't return an operation anymore, it shouldn't be called `operationFor...`. The bodies of these methods should probably me moved to the `sync...` methods that call them.